### PR TITLE
SPCTR-3526 Fix: Form - Resolved an issue wherein form submission not working, when ReCaptcha version is selected and enable ReCaptcha is disabled.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3526

### Screenshots

- https://d.pr/v/KRy73R

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Form - Resolved an issue wherein form submission not working, when ReCaptcha version is selected and enable ReCaptcha is disabled.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Form - Resolved an issue wherein form submissions would not work as intended when a ReCaptcha version is used and ReCaptcha is disabled.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update required.

File: blocks-config/forms/class-uagb-forms.php [hunk 2]
```diff
@@ -302,7 +302,7 @@ public function process_forms() {
 
 			}
 
-			if ( ! empty( $google_recaptcha_secret_key ) && ! empty( $google_recaptcha_site_key ) ) {
+			if ( is_array( $current_block_attributes ) && ! empty( $current_block_attributes['reCaptchaEnable'] ) && ! empty( $google_recaptcha_secret_key ) && ! empty( $google_recaptcha_site_key ) ) {
```
Comment (rahulvarma722) [inline blocks-config/forms/class-uagb-forms.php:305]: is_array( $current_block_attributes ) already verified in upper scope if not please verify in upper scope

**Review by rahulvarma722 — APPROVED** [review]

File: blocks-config/forms/class-uagb-forms.php (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/forms/class-uagb-forms.php]: Resolved

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**